components/global/portal/io.cellery.observability.ui/src/components/common/DependencyGraph.js

```diff
@@ -137,15 +141,23 @@ class DependencyGraph extends React.Component {
     };
 
     draw = () => {
-        const {nodeData, edgeData, onClickNode, onClickGraph, selectedCell, viewGenerator, graphType} = this.props;
+        const {nodeData, edgeData, onClickNode, onClickGraph, selectedCell,
+            viewGenerator, graphType, loadCellIngress} = this.props;
         const dataNodes = [];
         const dataEdges = [];
+        let label = "";
 
         if (nodeData) {
             nodeData.forEach((node, index) => {
+                if (graphType === DependencyGraph.GraphType.OVERVIEW) {
```

Comment:
This logic should not be handled by this component. The parent component whose passing the "loadCellIngress" function should handle it. Please remove "graphType" from props as well.